doc/Programming-guide/Key-platform-features/Data-collection/Rest-log-appender/index.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: page
+title: Rest log appender
+permalink: /:path/
+nav: /:path/Programming-guide/Key-platform-features/Data-collection/Rest-log-appender
+sort_idx: 90
+---
+
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+
+
+* TOC
+{:toc}
+
+The REST log appender is responsible for transferring logs from Operation server to your custom service.
+
+# Creating REST log appender with Admin UI
+
+The easiest way to create a Cassandra log appender for your application is by using Admin UI.
+
+To create a log appender for access via REST, do the following:
```

Comment:
Could this be re-written to something like below?

To create a REST log appender, do the following: